.github/workflows/ci.yml

```
@@ -85,26 +91,31 @@ jobs:
-
-      - name: Run VS Code integration tests (Linux)
-        if: runner.os == 'Linux'
-        run: xvfb-run -a npm test
+        run: |
+          echo "🧪 Running unit tests on Linux..."
+          # Run tests and capture result, ignore Mocha cleanup exit code 7
+          set +e  # Disable exit on error temporarily
+          xvfb-run -a ./node_modules/.bin/mocha --require out/test/shared/TestSetup.js --timeout 30000 'out/test/unit/**/*.test.js' --reporter spec
+          TEST_EXIT_CODE=$?
+          set -e  # Re-enable exit on error
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)